src/app/components/navbar/NavBar.jsx

```diff
@@ -8,34 +10,32 @@ import cookies from "../../utilities/cookies";
 import PreviewNav from "../preview-nav";
 
 const LANG = {
-    "en": "English",
-    "cy" : "Welsh"
-}
+    en: "English",
+    cy: "Welsh",
+};
```

Comment:
I can probably delete this now right if I have languageConstants?

Reply:
I would probably leave it as it is a dictionary for yours abbreviations. Maybe we don't need abbreviations?